### PR TITLE
renamed "SendRGBToString" method to "SendRGBToLED", edited the SHOOTERSPEED variables and where they are, removed flashlight calls, renamed to IsAtDesiredSpeed(), differentiated m_flywheelTargetRPM between lower and upper hub, set inversion to true on shooter motor

### PR DESCRIPTION
**Pull request by amull23**

### CODE REVIEW

**Review by cgjyang — APPROVED** [review]
Everything looks good 👍

File: Comp2022/src/main/cpp/commands/AutoShoot.cpp [hunk 1]
```diff
@@ -38,7 +38,7 @@ AutoShoot::AutoShoot(
     AddCommands(
         IntakeDeploy(true),
         AutoWait(drivetrain),
-        ShooterRun(Shooter::SHOOTERSPEED_FORWARD, shooter),
+        ShooterRun(Shooter::SHOOTERSPEED_LOWHUB, shooter),
```
Comment (cgjyang) [inline Comp2022/src/main/cpp/commands/AutoShoot.cpp:41]: Is the purpose of making the setting shooting at low hub at the moment just for testing? Just wondering

**Review by cgjyang — APPROVED** [review]
Looks good! Just have a question below

**Review by ns4robotics — APPROVED** [review]
Looks good 👍

**Pull request merged**